src/actions/mod.rs

```diff
@@ -35,6 +35,21 @@ use std::time::Duration;
 
 use self::compiler_message_parsing::{FileDiagnostic, ParseError, Suggestion};
 
+// TODO: Support non-`file` URI schemes in VFS. We're currently ignoring them because
+// we don't want to crash the RLS in case a client opens a file under different URI scheme
+// like with git:/ or perforce:/ (Probably even http:/? We currently don't support remote schemes).
+macro_rules! ignore_non_file_uri {
+    ($expr: expr; $log_name: expr; $uri: expr) => {
```

Comment:
I actually can. Previously when I was working on macros I encountered some ambiguities when separating `expr`s with commas, but it's fine here. Updated